catalog/app/containers/Bucket/PackageCopyDialog.js

```diff
@@ -178,9 +200,28 @@ function DialogForm({
         setNameWarning(warning)
       }
     },
-    [successor, nameExistence, nameWarning],
+    [nameWarning, nameExistence, successor],
   )
 
+  const [editorElement, setEditorElement] = React.useState()
+
+  const onFormChange = React.useCallback(
+    async ({ values }) => {
+      if (document.body.contains(editorElement)) {
+        setMetaHeight(editorElement.clientHeight)
+      }
+
+      handleNameChange(values.name)
+    },
+    [editorElement, handleNameChange, setMetaHeight],
+  )
+
+  React.useEffect(() => {
+    if (document.body.contains(editorElement)) {
```

Comment:
shouldnt we check if `editorElement` is not null?

Reply:
No, because `document.body` doesn't contain null :)
```js
document.body.contains(null) === false
```